### ISSUE TITLE
some GPUs missed in `nvidia-smi`

### ISSUE THREAD

**Issue by dyhBUPT** [NVIDIA/gpu-operator#643]
Hi, my device is `ubuntu1~18.04`, which has 7 Tesla T4 GPUs. It has worked well for over 2 years.
However, in this morning, 2 GPUs disappeared in `nvidia-smi`.
Then I reboot the device, and they reappeared.
But now, only after a few hours, 1 GPU disappears again, then another GPU disappears too.
So, what can I do for this?
Looking forward to your reply~
![image](https://github.com/NVIDIA/gpu-operator/assets/99722489/b73840de-2901-42c6-a9a9-980db46875a9)

**Comment by cdesiniotis** [comment]
@dyhBUPT I am going to close this issue since this does not appear related to the GPU Operator. If there is indeed a GPU Operator issue, feel free to re-open this issue or create another issue.

If you are still facing problems, I would advise opening a thread in the NVIDIA developer forums following this guidance: https://forums.developer.nvidia.com/t/if-you-have-a-problem-please-read-this-first/27131
